rest-api-spec/src/main/resources/rest-api-spec/test/cat.aliases/10_basic.yaml

```diff
@@ -127,6 +127,37 @@
       $body: / (^|\n)test_2 .+ \n/
 
 ---
+"Multiple alias names":
+
+  - do:
+        indices.create:
+            index: test
+
+  - do:
+        indices.create:
+            index: test2
+
+  - do:
+        indices.put_alias:
+            index: test
+            name: foo
+
+  - do:
+        indices.put_alias:
+            index: test2
+            name: bar
+
+  - do:
+        cat.aliases:
+            name: foo,bar
+
+  - match:
+      $body: /(^|\n)foo\s+test /
+
+  - match:
+      $body: /(^|\n)bar\s+test2 /
+
```

Comment:
I think that this test should add a third index, and a third alias, and not request that alias (as you've already done) just so that we can be sure that a non-requested alias is not returned.